tmt/steps/provision/testcloud.py

```diff
@@ -201,6 +199,23 @@ class TestcloudGuestData(tmt.steps.provision.GuestSshData):
     instance_name: Optional[str] = None
     list_local_images: bool = False
 
+    # TODO: custom handling for two fields - when the formattign moves into
+    # field(), this should not be necessary.
+    def show(
+            self,
+            *,
+            keys: Optional[List[str]] = None,
+            verbose: int = 0,
+            logger: tmt.log.Logger) -> None:
+
+        keys = keys or list(set(self.keys()))
```

Comment:
Do you expect that `self.keys()` can return duplicate key names?

Reply:
Fixed in 50e02b73

Reply:
> Fixed in [50e02b7](https://github.com/teemtee/tmt/commit/50e02b73e84668531c6d8b6e4d41607a0119ad32)

That one actually changed it in `steps/provision/__init__.py`, I've covered the `testcloud.py` part as well and squashed.